egcg_core/util.py

```diff
@@ -97,5 +89,6 @@ def move_dir(src_dir, dest_dir):
                 dest_file = os.path.join(dest_dir, os.path.basename(fp))
                 shutil.move(fp, dest_file)
         return 0
-    except OSError:
+    except OSError as e:
+        app_logger.error('OSError: ' + str(e))
```

Comment:
It would be easier to say if we knew exactly what raises this error.
The only reference to OSError I can find is in https://bugs.python.org/issue1438480 which point to permission issue when the owner of the file is not production.